packages/flutter/lib/src/services/autofill.dart

```diff
@@ -0,0 +1,273 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'text_input.dart';
+
+// TODO(LongCatIsLooong): expand the list to include every predefined value.
+/// A collection of commonly used autofill hint strings on different platforms.
+class AutofillHints {
```

Comment:
Could this be an enum?

Reply:
It seems all 3 platforms use strings as autofill hints. Why is enum more suitable in this case?

Reply:
Sounds good.